src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -2888,6 +2888,30 @@
+    ("SveLoadMaskedUnOpTest.template",    new Dictionary<string, string> {["TestName"] = "SveLoadVectorSByteSignExtendToUInt64",                                                                   ["Isa"] = "Sve",                                   ["Method"] = "LoadVectorSByteSignExtendToUInt64",                                    ["RetVectorType"] = "Vector",    ["RetBaseType"] = "UInt64",  ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "UInt64",                                  ["Op2BaseType"] = "SByte",                                                               ["LargestVectorSize"] = "8",                                                                ["NextValueOp2"] = "TestLibrary.Generator.GetSByte()",                                                                                         ["ValidateIterResult"] = "(ulong)firstOp[i] != result[i]"}),
+    ("SveLoadMaskedUnOpTest.template",    new Dictionary<string, string> {["TestName"] = "SveLoadVectorUInt16ZeroExtendToInt32",                                                                   ["Isa"] = "Sve",                                   ["Method"] = "LoadVectorUInt16ZeroExtendToInt32",                                    ["RetVectorType"] = "Vector",    ["RetBaseType"] = "Int32",   ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "Int32",                                   ["Op2BaseType"] = "UInt16",                                                              ["LargestVectorSize"] = "8",                                                                ["NextValueOp2"] = "TestLibrary.Generator.GetUInt16()",                                                                                        ["ValidateIterResult"] = "firstOp[i] != result[i]"}),
+    ("SveLoadMaskedUnOpTest.template",    new Dictionary<string, string> {["TestName"] = "SveLoadVectorUInt16ZeroExtendToInt64",                                                                   ["Isa"] = "Sve",                                   ["Method"] = "LoadVectorUInt16ZeroExtendToInt64",                                    ["RetVectorType"] = "Vector",    ["RetBaseType"] = "Int64",   ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "Int64",                                   ["Op2BaseType"] = "UInt16",                                                              ["LargestVectorSize"] = "8",                                                                ["NextValueOp2"] = "TestLibrary.Generator.GetUInt16()",                                                                                        ["ValidateIterResult"] = "firstOp[i] != result[i]"}),
```

Comment:
This is the comment for `LoadVector` as well - but it seems that the `mask` vector gets populated with `"TestLibrary.Generator.Get*()` which is mostly non-zero. We should also add a way to have in-active elements in the mask and make sure that they remain untouched.

Reply:
>We should also add a way to have in-active elements in the mask and make sure that they remain untouched.

Agreed. the tricky part of creating the mask and knowing which elements are set. We could use the API methods, but most haven't been implemented yet.

Alternatively - add a helper which constructs a Vector<T> filled randomly with 0 and 1 elements. The implicit mask to vector conversions mean that this can be used for the load mask. Then validateResult needs passing the mask so it can check the 0 entries in the mask match to 0s in the result.